src/jsonpatcherproxy.js

```diff
@@ -143,7 +144,9 @@ var JSONPatcherProxy = (function() {
         return Reflect.deleteProperty(target, key);
       }
     });
-    return proxy;
+    /* keeping track of all the proxies to be able to revoke them later */
+    this.proxifiedObjectsMap.push(proxy);
+    return proxy.proxy;
   };
   //grab tree's leaves one by one, encapsulate them into a proxy and return
   JSONPatcherProxy.prototype._proxifyObjectTreeRecursively = function(
```

Comment:
@tomalec now I implemented the ability to disable traps for a given proxy, [and this happens automatically in `deleteProperty` trap](https://github.com/Palindrom/JSONPatcherProxy/pull/4/commits/b1022e8336e256f0f448770987ad12e12a07296f#diff-3bfa726203c6cb018f2321c18ca9298eR91). It's more fail-proof. And I regret the revoking proposal, it's redundant now. I'll explain:

If an element (say an HTML `input` field) is bound to as sub-object of the state tree using Polymer binding. And a patch comes to delete that sub-object from the server, editing the field will throw a TypeError (in the time window of Polymer binding async update), and this error will be to the user in production, not to the developer. Any race condition will cause this issue and it will be very hard to reproduce. 

WDYT?